about.html

```diff
@@ -0,0 +1,208 @@
+        </li>
+        <li class="nav-list">
+          <a class="nav3" href="#">Join</a>
+        </li>
+        <li class="nav-list">
+          <a class="nav4" href="#">Sponsor</a>
+        </li>
+        <li class="nav-list">
+          <a class="nav5" href="#">News</a>
+        </li>
+        <li class="nav-list">
+          <a class="nav6" href="#">Campaign</a>
+        </li>
+      </ul>
+    </nav>
+
+    <div class="about-section-main">
+      <h2 class="headline">Welcome to the Institute of</h2>
+      <h3 class="hl2">Culinary Education</h3>
+      <h4 class="hl3">Ranked No.1 Culinary School in South America*</h4>
+
+      <p class="para1-about">
+        The Institute of Culinary Education is a world-class culinary school
+        located in the heart of New York City. We offer a variety of programs to
+        help you achieve your culinary goals, whether you’re looking to become a
+        professional chef, start a new career, or just expand your cooking
+        skills.
+      </p>
+
+      <p class="about-about-page">
+        Please contact us by Email for any further questions about The Institute
+        of Culinary Education 2022!
+      </p>
+      <h2 class="about-email">
+        <a class="about-orange" href="mailto:admissions@ice.org"
+          >Email us at admissions@ice.org</a
+        >
+      </h2>
+
+      <h2 class="about-lower">About the Institute of Culinary Education</h2>
+      <div class="line"></div>
+
+      <p class="about-discuss">
+        Award-winning, vibrant and diverse, the Institute of Culinary Education
+        has helped thousands of students find their culinary voices — and we're
+        ready to help you find yours. With campuses in dynamic and exciting food
+        cities — New York City and Los Angeles — ICE works at the forefront of
+        the food and hospitality industries.
+      </p>
+
+      <img src="logo.svg" alt="ice" class="ice-img" />
+    </div>
+    <hr class="about-hr" />
+
+    <section class="about-botttom-section">
+      <div>
+        <h2 class="graduates">Look at some of our past graduates</h2>
+        <p class="line"></p>
+        <p class="more-info-about">
+          Institute of Culinary Education has helped thousands of students find
+          their culinary voices — and we're ready to help you find yours.
+        </p>
+      </div>
+    </section>
+
+    <section class="photos-about">
+      <div class="past-graduates">
+        <img src="aboutchef3.jpg" alt="graduates" class="aboutchef-img" />
+        <div class="aboutchef">
+          Chef Johan Sanchez Graduation class of 2021 <br />
+          Now Head Chef at BlueBird
+        </div>
+      </div>
+      <div class="past-graduates">
+        <img src="aboutchef2.jpg" alt="graduates" class="aboutchef-img2" />
+        <div class="aboutchef">
+          Chef Andres Gomez Graduation class of 2021 <br />
+          Now Head Chef at Mestizo
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] in the `past graduates` section in the mobile view the 2 pictures should be displayed as a column instead of a row. Kindly fix this issue 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/191625015-1500b851-6e60-467b-9a2b-17bddf7b65d7.png)

## Design guidelines:
![image](https://user-images.githubusercontent.com/50721479/191625035-6fbcf277-6847-4857-984d-f22885be58df.png)

Reply:
ok this is now fixed